components/server/src/prometheus-metrics.ts

```diff
@@ -30,7 +30,15 @@ const loginCounter = new prometheusClient.Counter({
     labelNames: ["status", "auth_host"],
 });
 
-export function increaseLoginCounter(status: string, auth_host: string) {
+type LoginCounterStatus =
+    // The login attempt failed due to a system error (picked up by alerts)
+    | "failed"
+    // The login attempt succeeded
+    | "succeeded"
+    // The login attempt failed, because the client failed to provide complete session information, for instance.
+    | "failed_client";
```

Comment:
`user_error`? Feels like it's inventing a new terminology

Reply:
I started out with user, but I felt wrong because we're still before the login phase. This is more about the technical level and the client behaving correctly.

Reply:
How is the client behaving correctly? In this case, they supplied the wrong auth flow so it's an Invalid Argument - Category of User Errors.